web-app/src/make_pickle.py

```diff
@@ -0,0 +1,48 @@
+import pickle
```

Comment:
It may look a bit absurd that the command-line utility `make_pickle` is next to `make_celery`, but when I tried to put it into a separate `utils` directory, I struggled with Python package layout and relative imports

Reply:
hmm in the tutorial the whole web app was designed as python package with a top level _init__.py to start the webservice. Maybe I should have followed this to avoid such problems